### DISCUSSION TITLE
Tailwind not loading with tailwind.config.ts

### DISCUSSION THREAD

**Discussion by Victor-Ifeel** [tailwindlabs/tailwindcss#11083] [Help]
Tailwind does not load CSS after upgrading from 3.1.8 to 3.3.1 and changing the configuration from tailwind.config.js to tailwind.config.ts

The config file seems correct correct when I run this command it generates the file correctly
`npx tailwindcss -i ./src/index.css -o ./build/output.css`

But it does not work when I make a `npm run start`

This is the config file

> import type {Config} from 'tailwindcss';
> 
> export default {
>   content: ['./src/**/*.{js,jsx,ts,tsx}'],
>   theme: {
>     borderRadius: {
>       none: '0',
>       sm: '4px',
>       DEFAULT: '4px',
>       md: '6px',
>       lg: '0.5rem',
>       full: '9999px',
>       large: '12px',
>       card: '20px',
>     },
>     extend: {
>       boxShadow: {
>         banner: '0px 0px 20px rgba(195, 203, 219, 0.24)',
>         bubble: '0px 2px 3px rgba(162, 170, 184, 0.2)',
>         conversation: '0px 3px 4px #EFECEC',
>         emoji: '0px 0px 20px rgb(195 203 219 / 24%)',
>         card: '0px 12px 40px rgb(117 138 180 / 20%)',
>         journey: '0px 4px 20px rgb(172 175 189 / 19%)',
>         menu: '0px 1px 10px rgba(121, 123, 129, 0.1)',
>         modal: '0px 1px 10px #E9ECF3',
>         header: '0px 1px 2px rgba(0, 0, 0, 0.1) ',
>         shadow1: '0px 1px 2px rgba(0, 0, 0, 0.1)',
>         shadow2: '0px 2px 6px rgba(0, 0, 0, 0.1)',
>         shadow3: '0px 3px 10px rgba(0, 0, 0, 0.1)',
>       },
>       colors: {
>         semantic: {
>           link: {
>             500: '#2789FD',
>           },
>         },
>         text: {
>           10: '#F0F000',
>           50: '#F0F000',
>           100: '#F0F000',
>           300: '#F0F000',
>           400: '#F0F000',
>           500: '#F0F000',
>           600: '#F0F000',
>           700: '#F0F000',
>         },
>         textCold: {
>           500: '#F0F000',
>         },
>         textWarm: {
>           100: '#F0F000',
>           400: '#F0F000',
>           700: '#F0F000',
>         },
>         brandPrimary: {
>           50: '#F0F000',
>           100: '#F0F000',
>           200: '#F0F000',
>           400: '#F0F000',
>           600: '#F0F000',
>           700: '#F0F000',
>           800: '#F0F000',
>         },
>         brandPrimaryLight: {
>           200: '#F0F000',
>           300: '#F0F000',
>           400: '#F0F000',
>           500: '#F0F000',
>         },
>         brandSecondary: {
>           100: '#F0F000',
>           200: '#F0F000',
>           400: '#F0F000',
>           700: '#F0F000',
>         },
>         brandSalmon: {
>           50: '#F0F000',
>           300: '#F0F000',
>           400: '#F0F000',
>           500: '#F0F000',
>           600: '#F0F000',
>           700: '#F0F000',
>         },
>         brandComplementaryAmarillo: '#F0F000',
>         brandComplementaryAquaMarina: '#F0F000',
>         backgrounds: {
>           1: '#F0F000',
>           2: '#F0F000',
>           3: '#F0F000',
>           4: '#F0F000',
>           5: '#F0F000',
>           6: '#F0F000',
>           7: '#F0F000',
>           8: '#F0F000',
>         },
>         borders: {
>           conversation: '#F0F000',
>         },
>         danger: {
>           500: '#F0F000',
>           700: '#F0F000',
>         },
>         gradientFrom: {
>           1: '#F0F000',
>           2: '#F0F000',
>           3: '#F0F000',
>           4: '#F0F000',
>           5: '#F0F000',
>         },
>         gradientTo: {
>           1: '#F0F000',
>           2: '#F0F000',
>           3: '#F0F000',
>           4: '#F0F000',
>           5: '#F0F000',
>         },
>         overLay: '#F0F000',
>         success: {
>           100: '#F0F000',
>           300: '#F0F000',
>           500: '#F0F000',
>           700: '#F0F000',
>         },
>         warning: {
>           500: '#F0F000',
>           700: '#F0F000',
>         },
>         links: '#F0F000',
>         avatar: '#F0F000',
>       },
>       dropShadow: {
>         menu: '0px 1px 10px rgba(121, 123, 129, 0.1)',
>       },
>       fontFamily: {
>         recoleta: ['Recoleta', 'serif'],
>         sofia: ['Sofia', 'sans-serif'],
>         'opens-sans': ['Open Sans', 'sans-serif'],
>       },
>       fontSize: {
>         body14: ['14px', '18px'],
>         body16: ['16px', '22px'],
>         lg: ['20px', '28px'],
>         xl: ['24px', '32px'],
>       },
>       fontWeight: {
>         regular: '400',
>       },
>       height: {
>         avatarCard: '43px',
>         avatarCardConversation: '60px',
>         moodPulse: '150px',
>       },
>       padding: {
>         card: '20px',
>         sidebar: '338px',
>       },
>       maxWidth: {
>         page: '924px',
>         'page-md': '620px',
>         patientsPage: '1240px',
>       },
>       width: {
>         avatarCard: '43px',
>         avatarCardConversation: '60px',
>         moodPulse: '150px',
>       },
>     },
>   },
>   plugins: [],
> } satisfies Config;
>

**Comment by wongjn** [comment]
Consider providing a remotely-hosted git repository that reproduces the unexpected behavior.
